doc/sphinx/source/tutorials/intermediate/create_radio.rst

```diff
@@ -0,0 +1,90 @@
+
+Before starting, you have to bind your radio to the receiver, and configure your radio to send all needed channels. See the documentation of your radio and receiver to do it. For some radios, there is nothing to configure, but for others, you will have to select the channels you want to send. The configuration depend on your radio so it will not be explained here, but you will be able to know if all needed channels are send.
+
+Fist, power-on your radio and your drone, and start a session with the needed processes to fly a drone : you should at least launch the *Server*, the *Data Link* and the *GCS*.
+
+Wait to have link with the drone, then set the telemetry mode to *ppm*.
+
+To change the telemetry mode, go on the *Settings* tab, then *System* and *Telemetry*. Select *ppm* in the drop down box and send this command to the drone by hitting the commit button. Wait for the drone to be in *ppm* mode. (The label at the left of the drop down box should change to *ppm* shortly after hitting the commit button.)
+
+.. image:: ppm_mode.png
+    :alt: Set telemetry in PPM mode.
+    :align: center
+
+Once you are in *ppm* mode, go to the paparazzi center, then launch the "Messages" process (*Tools* -> *Messages*).
+
+You should see a *PPM* section. Click on it to display the values, shown coma separated. If you move the joystick on your radio, the values should change. Check if all needed axis are working, including mode switch and kill switch. If this is not he case, the problem is most probably coming from your radio.
+
+.. image:: ppm_messages.png
+    :alt: PPM messages
+    :align: center
+
+Go in ``$PAPARAZZI_SRC/conf/radio`` and copy the ``dummy.xml`` file with the name of your radio. Edit it, and modify the ``name`` attribute of the ``radio`` tag with your radio name.
+Add as many channel as wanted, based on the following example :
+
+.. code-block:: xml
+
+    <radio name="My own FrSky X9D" data_min="900" data_max="2100" sync_min ="5000" sync_max ="15000" pulse_type="POSITIVE">
+        <channel ctl="D" function="ROLL"     min="987" neutral="1503" max="2011" average="0"/>
+        <channel ctl="C" function="PITCH"    min="2011" neutral="1451" max="987" average="0"/>
+        <channel ctl="B" function="YAW"      min="990" neutral="1500" max="2011" average="0"/>
+        <channel ctl="A" function="THROTTLE" min="998" neutral="998" max="2011" average="0"/>
+        <channel ctl="PHAUX2" function="MODE"    min="2011" neutral="1500" max="987" average="1"/>
+    </radio>
+
+You should at least have these four channels to control your drone. The last one is for the mode selection.
+
+
+
+It is essential that the order of these lines match the order of the channels in the *Messages* process.
+
+For each channel, move the joystick to its minimum, neutral and maximum position, and look at the values displayed in the *Messages* process. Write these values in the ``min``, ``neutral`` and ``max`` attibutes of the corresponding channel in your radio file.
```

Comment:
I have made some changes here with the new configuration system (in master, not yet in a stable branch):
- you can use the `reverse` attribute instead of swapping min and max values
- the `ctl` is not mandatory and even deprecated (it has never been used anyway)

But this is completely backward compatible and the old-way is still fine. We can wait for the next stable release to update this maybe.

Reply:
ok, edits done